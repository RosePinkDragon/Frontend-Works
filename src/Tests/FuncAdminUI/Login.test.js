import { mount } from "enzyme";
import LoginScreen from "components/Tasks/AdminUI/Login";

describe("log's out onClick", () => {
  let data = false;

  const logger = (change) => {
    data = change;
  };

  let wrapper = mount(<LoginScreen logger={logger} />);

  it("should render the Login Screen Component correctly", () => {
    expect(wrapper).toMatchSnapshot();
  });

  it("changes the logger", () => {
    wrapper.find("form").simulate("submit");
    expect(data).toEqual(true);
  });
});
