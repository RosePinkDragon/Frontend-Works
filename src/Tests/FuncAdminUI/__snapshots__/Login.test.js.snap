// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`log's out onClick should render the Login Screen Component correctly 1`] = `
<LoginScreen
  logger={[Function]}
>
  <div
    className="login"
  >
    <form
      onSubmit={[Function]}
    >
      <div
        className="formElement"
      >
        <label
          htmlFor="email"
        >
          Email
        </label>
        <input
          id="email"
          name="email"
          required={true}
          type="email"
        />
      </div>
      <div
        className="formElement"
      >
        <label
          htmlFor="password"
        >
          Password
        </label>
        <input
          id="password"
          name="password"
          required={true}
          type="password"
        />
      </div>
      <button
        className="btn"
      >
        login
      </button>
    </form>
  </div>
</LoginScreen>
`;
